src/main/java/com/jmb/springfactory/controller/GenericController.java

```diff
@@ -1,8 +1,16 @@
 package com.jmb.springfactory.controller;
 
 import java.io.Serializable;
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)